solutions/usaco-598.mdx

```diff
@@ -2,11 +2,16 @@
 id: usaco-598
 source: USACO Gold 2016 January
 title: Radio Contact
-author: Neo Wang
+author: Neo Wang, Melody Yu (Video)
```

Comment:
```suggestion
author: Neo Wang, Melody Yu
```
idt it's necessary to specify video or language in the authors section (can you also remove from the other prs?)